pkg/storage/admitters/backup.go

```diff
@@ -220,3 +233,79 @@ func validatePVCNameExists(vmBackup *backupv1.VirtualMachineBackup, causes []met
 	}
 	return causes
 }
+
+// VMBackupTrackerAdmitter validates VirtualMachineBackupTrackers
+type VMBackupTrackerAdmitter struct {
+	Config *virtconfig.ClusterConfig
+}
+
+// NewVMBackupTrackerAdmitter creates a VMBackupTrackerAdmitter
+func NewVMBackupTrackerAdmitter(config *virtconfig.ClusterConfig) *VMBackupTrackerAdmitter {
+	return &VMBackupTrackerAdmitter{
+		Config: config,
+	}
+}
+
+// Admit validates an AdmissionReview for VirtualMachineBackupTracker
+func (admitter *VMBackupTrackerAdmitter) Admit(ctx context.Context, ar *admissionv1.AdmissionReview) *admissionv1.AdmissionResponse {
+	if ar.Request.Resource.Group != backupv1.SchemeGroupVersion.Group ||
+		ar.Request.Resource.Resource != "virtualmachinebackuptrackers" {
+		return webhookutils.ToAdmissionResponseError(fmt.Errorf("unexpected resource %+v", ar.Request.Resource))
+	}
+
+	if ar.Request.Operation == admissionv1.Create && !admitter.Config.IncrementalBackupEnabled() {
```

Comment:
obviously need a webhook to check the featurgate but I think it would be good to do other checks in CEL statements in the CRD if possible. I think you can enforce spec immutability that way for example